ui/app/abilities/abstract.js

```diff
@@ -12,19 +12,24 @@ export default class Abstract extends Ability {
   @not('token.aclEnabled') bypassAuthorization;
   @equal('token.selfToken.type', 'management') selfTokenIsManagement;
 
-  @computed('system.activeNamespace.name')
-  get activeNamespace() {
-    return this.get('system.activeNamespace.name') || 'default';
+  // Pass in a namespace to `can` or `cannot` calls to override
+  // https://github.com/minutebase/ember-can#additional-attributes
+  namespace = 'default';
+
+  get _namespace() {
+    if (!this.namespace) return 'default';
+    if (typeof this.namespace === 'string') return this.namespace;
+    return get(this.namespace, 'name');
   }
```

Comment:
`activeNamespace` was previously set in our models based on query parameters. We're noticing that we can't [detect namespace-specific policy capabilities](https://github.com/hashicorp/nomad/issues/10885) (with ACL's).

Should we continue the behavior of setting `system.activeNamespace` in our models to grab namespaces from the query params.

Also, since we're setting `namespace` to `default` on line 17, what's the purpose of the control flow logic on lines 19-22.

Reply:
Maybe we can set this value based on `qpNamespace`